ci/packaging/suse/skuba_spec_template

```diff
@@ -60,7 +60,7 @@ Summary:        Utility to automatically refresh and update a skuba cluster
 Group:          System/Management
 Requires:       python3-setuptools
 Requires:       zypper >= 1.14.15
-Requires:       kubernetes-client
```

Comment:
To prevent the upgrade of the client without reason, we should _not_ do this.

Instead we should remove the requirement, and be explicit about possible breakages.

Reply:
If the removal breaks the greenfields's skuba-update (which I doubt we have test coverage for), then we can probably iterate with `Requires: (kubernetes-client or k8s-client)`

Reply:
The Requires of (kubernetes-client or k8s-client) triggers the install of kubernetes-1.18-client during zypper dup, so I removed this require.